ephios/plugins/complexsignup/structure.py

```diff
@@ -41,22 +45,21 @@ def atomic_block_participant_qualifies_for(structure, participant: AbstractParti
 
 
 def _build_human_path(structure):
-    return " » ".join(
-        [
-            *[s["display"] for s in reversed(structure["parents"])],
-            f"{structure['display']} #{structure['number']}",
-        ]
-    )
+    # put the atomic block display name first, in case the whole path gets cut off
+    if structure["display"] != structure["name"]:
+        s = f"{structure['display']} - {structure['name']} #{structure['number']}"
```

Comment:
is there a reason why this is not part of the display string already?

Reply:
tbh I was too lazy to change the display value.